model-optimizer/setup.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+"""
+ Copyright (C) 2018-2020 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+"""
+Use this script to create a wheel with Model Optimizer code:
+
+$ python setup.py sdist
+$ python -m pip wheel --no-index --no-deps --wheel-dir dist dist/*.tar.gz
+"""
+
+import sys
+import os
+import re
+from setuptools import setup, find_packages
+from setuptools.command.install import install
+from setuptools.command.build_py import build_py
+
+package_name = 'mo'
+
+# Detect all the framework specific requirements_*.txt files.
+requirements_txt = []
+py_modules = []
+for name in os.listdir():
+    if re.match('requirements_(.*)\.txt', name):
+        requirements_txt.append(name)
+    if re.match('mo_(.*)\.py', name):
+        py_modules.append(name.split('.')[0])
+
+# Minimal set of dependencies
+deps = [
+    'networkx>=1.11',
+    'defusedxml>=0.5.0',
+    'numpy>=1.14.0',
+]
```

Comment:
I would recommend to read the exact version conditions of these components from the `requirements.txt` file during building the wheel. So, let's hardcode the minimal list of component names and write a function which will get a version condition from the `requirements.txt` file

Reply:
Good point! Fixed